.github/workflows/release.yml

```diff
@@ -18,28 +52,33 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install Rust
-        run: rustup update 1.67.1 --no-self-update && rustup default 1.67.1
+        run: rustup update stable --no-self-update && rustup default stable
```

Comment:
did you do this on purpose?

Reply:
i did- mostly because i knew i would forget to and wanted to pin to stable